### PR TITLE
[MB-1346] Refactored code in RDBMSConstants

### PR DESCRIPTION
**Pull request by sdkottegoda**
Addresses https://wso2.org/jira/browse/MB-1346

### CODE REVIEW

File: modules/andes-core/broker/src/main/java/org/wso2/andes/store/rdbms/RDBMSConstants.java [hunk 1]
```diff
@@ -146,316 +146,300 @@
 
     // prepared statements for Message Store
     protected static final String PS_INSERT_MESSAGE_PART =
-            "INSERT INTO " + CONTENT_TABLE + "(" +
-                    MESSAGE_ID + "," +
-                    MSG_OFFSET + "," +
-                    MESSAGE_CONTENT + ") " +
-                    "VALUES (?, ?, ?)";
-
-    protected static final String PS_DELETE_MESSAGE_PARTS =
```
Comment (hemikak) [inline modules/andes-core/broker/src/main/java/org/wso2/andes/store/rdbms/RDBMSConstants.java]: This sql is not needed anymore ?

Comment (sdkottegoda) [inline modules/andes-core/broker/src/main/java/org/wso2/andes/store/rdbms/RDBMSConstants.java]: No. We don't explicitly delete message parts any more.

**Comment by a5anka** [comment]
:+1:

**Comment by ramith** [comment]
+1

File: modules/andes-core/broker/src/main/java/org/wso2/andes/store/rdbms/RDBMSConstants.java (hunk 1 above)
Comment (ramith) [inline modules/andes-core/broker/src/main/java/org/wso2/andes/store/rdbms/RDBMSConstants.java]: shall we remove.?

Comment (sdkottegoda) [inline modules/andes-core/broker/src/main/java/org/wso2/andes/store/rdbms/RDBMSConstants.java]: It is removed by this PR.

**Pull request merged**